tests/installation/install_11SP4_preparation.pm

```diff
@@ -0,0 +1,43 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Remove gnome and add kde pattern for migration from SLES11SP4+kde
+# - For test scenario of migration from SLES11SP4+kde to SLES15SP3, 
+# gnome pattern is selected as default, we need remove gnome environment pattern 
+# then add kde environment pattern before start installation.
+# Maintainer: Lemon Li <leli@suse.de>
+
+use base 'y2_installbase';
+use strict;
+use warnings;
+use lockapi;
+use testapi;
+use mmapi;
+use version_utils qw(is_sle is_upgrade);
+
+sub run {
+    my ($self) = @_;
+    if (check_var('DESKTOP', 'kde')) {
+        send_key_until_needlematch 'packages-section-selected', 'tab';
+        send_key 'ret';
+        wait_still_screen 1;
```

Comment:
you might have misread my suggestion in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/11031#discussion_r491909830 . An explicit `assert_screen` is in most cases much more efficient than the time waste of `wait_still_screen 1`. Unless you can state a very good reason for why `wait_still_screen` should be preferred please use `assert_screen`

Reply:
Yes, I think I misunderstanding your suggestion. Just updated code with assert_screen to sync. Thanks.

Reply:
Verification log:  https://openqa.nue.suse.com/tests/4710403#step/install_11sp4_preparation/5